packages-content-model/roosterjs-content-model-types/lib/format/formatParts/EntityInfoFormat.ts

```diff
@@ -16,4 +16,9 @@ export type EntityInfoFormat = {
      * Type of this entity
      */
     entityType?: string;
+
+    /**
+     * Whether this entity is a block entity
+     */
+    isBlock?: boolean;
```

Comment:
Question: do we really need to know if an entity is block now? What if we just always remember its style like width and display, then just write them back as it is, so we always treat the entity as inline?

Reply:
Discussed with Jason, I added this because on paste browser removed the percentage and set px width. 

We will remove this and move it to only copy and paste handlers